cmd/vela-server/main.go

```diff
@@ -60,6 +60,21 @@ func main() {
 			Value:   false,
 		},
 
+		// Security Flags
+
+		&cli.DurationFlag{
+			EnvVars: []string{"VELA_ACCESS_TOKEN_DURATION", "ACCESS_TOKEN_DURATION"},
+			Name:    "access-token-duration",
+			Usage:   "sets the duration of the access token",
+			Value:   15 * time.Minute,
```

Comment:
just wondering, is there any significance to choosing 15 minutes?

Reply:
none, it's just a default. easy to override. i think the point is just to make the token useful for a small time window in case it gets compromised.